docs/dev.rst

```diff
@@ -37,6 +97,147 @@ The Symbols Service webapp is at: http://localhost:3000
 The Symbolication Service webapp is at: http://localhost:8050
 
 
+Bugs / Issues
+=============
+
+All bugs are tracked in `Bugzilla <https://bugzilla.mozilla.org/>`_.
+
+Write up a new bug:
+
+https://bugzilla.mozilla.org/enter_bug.cgi?product=Tecken&component=General
+
+If you want to do work for which there is no bug, it's best to write up a bug
+first. Maybe the ensuing conversation can save you the time and trouble
+of making changes!
+
+
+Code workflow
+=============
+
+Bugs
+----
+
+Either write up a bug or find a bug to work on.
+
+Assign the bug to yourself.
+
+Work out any questions about the problem, the approach to fix it, and any
+additional details by posting comments in the bug.
+
+
+Pull requests
+-------------
+
+Pull request summary should indicate the bug the pull request addresses. For
+example::
+
+    bug nnnnnnn: removed frob from tree class
+
+Pull request descriptions should cover at least some of the following:
+
+1. what is the issue the pull request is addressing?
+2. why does this pull request fix the issue?
+3. how should a reviewer review the pull request?
+4. what did you do to test the changes?
+5. any steps-to-reproduce for the reviewer to use to test the changes
+
+After creating a pull request, attach the pull request to the relevant bugs.
+
+We use the `rob-bugson Firefox addon
+<https://addons.mozilla.org/en-US/firefox/addon/rob-bugson/>`_. If the pull
+request has "bug nnnnnnn: ..." in the summary, then rob-bugson will see that
+and create a "Attach this PR to bug ..." link.
+
+Then ask someone to review the pull request. If you don't know who to ask, look
+at other pull requests to see who's currently reviewing things.
+
+
+Code reviews
+------------
+
+Pull requests should be reviewed before merging.
+
+Style nits should be covered by linting as much as possible.
+
+Code reviews should review the changes in the context of the rest of the system.
+
+
+Landing code
+------------
+
+Once the code has been reviewed and all tasks in CI pass, the pull request
+author should merge the code.
+
+This makes it easier for the author to coordinate landing the changes with
+other things that need to happen like landing changes in another repository,
+data migrations, configuration changes, and so on.
+
+We use "Rebase and merge" in GitHub.
+
+
+Conventions
+===========
+
+Python code conventions
+-----------------------
+
+All Python code files should have an MPL v2 header at the top::
```

Comment:
I wonder if we can offload that to black. I'll see if that's possible and if so, create a bug for it. WDYT?

Reply:
ah, same for the JS one

Reply:
I think we have a license checker script/tool somewhere, but it's definitely not active or hooked up anywhere.

Reply:
I'll see if I can find it

Reply:
I hunted for a bit and then wrote up this bug with findings and next step:

https://bugzilla.mozilla.org/show_bug.cgi?id=1764622